CCDB/src/CcdbApi.cxx

```diff
@@ -216,6 +216,28 @@ void CcdbApi::init(std::string const& host)
 
   mNeedAlienToken = (host.find("https://") != std::string::npos) || (host.find("alice-ccdb.cern.ch") != std::string::npos);
 
+  // Set the curl timeout. It can be forced with an env var or it has different defaults based on the deployment mode.
+  if (getenv("ALICEO2_CCDB_CURL_TIMEOUT")) {
+    auto timeout = atoi(getenv("ALICEO2_CCDB_CURL_TIMEOUT"));
+    if (timeout >= 0) { // if valid int
+      mCurlTimeout = timeout;
+    }
+  } else { // set a default depending on the deployment mode
+    o2::framework::DeploymentMode deploymentMode = o2::framework::DefaultsHelpers::deploymentMode();
+    LOG(info) << "deploymentMode : " << static_cast<std::underlying_type<o2::framework::DeploymentMode>::type>(deploymentMode);
```

Comment:
Do we really need this info message? Can we make it debug as well?

Reply:
oups sorry, my bad, removing.